container_workflow_tool/utility.py

```diff
@@ -19,6 +19,8 @@ def print_help(self, file=None):
 
 # Utility textwrap functions
 def _2sp(a): return textwrap.indent(a, '  ')
+
+
```

Comment:
I guess this is forced by whatever style checker we are using right

Reply:
Yes, this is caught by style checker.